src/front/js/store/flux.js

```diff
@@ -1,46 +1,26 @@
 const getState = ({ getStore, getActions, setStore }) => {
 	return {
 		store: {
-			message: null,
-			demo: [
-				{
-					title: "FIRST",
-					background: "white",
-					initial: "white"
-				},
-				{
-					title: "SECOND",
-					background: "white",
-					initial: "white"
-				}
-			]
+			user: {},
+			email_test: "persefone@gmail",
+			base_url: "https://3001-gray-slug-nautwk8k.ws-eu08.gitpod.io/"
```

Comment:
pon la url como https://3001-gray-slug-nautwk8k.ws-eu08.gitpod.io/api/  así no tenéis que ponerlo todo el rato

Reply:
https://github.com/kholostyak17/project-solo-travel/blob/97aa3a20d1f85afa4adf6f920260e17708f8a9aa/src/front/js/store/flux.js#L10

El acceso a la "/api/" está añadido en el fetch porque cuando se copia-pega la api viene sin la extension de la ruta.